app/icons/ic-dropdown.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you don't need this duplicate declaration. You should able to reuse this icon from the library